### PR TITLE
Improve CircleCI benchmark job

### PR DESCRIPTION
**Pull request by mikz**
* store CSV report
* benchmark master branch on the same machine for comparison as there can be huge performance difference between machines
* `make benchmark`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -144,6 +144,16 @@ node_modules/.bin/markdown-link-check:
 test-doc: node_modules/.bin/markdown-link-check
 	@find . \( -name node_modules -o -name .git -o -name t \) -prune -o -name "*.md" -print0 | xargs -0 -n1  -I % sh -c 'echo; echo ====================; echo Checking: %; node_modules/.bin/markdown-link-check  %' \;
 
+benchmark: export IMAGE_TAG ?= master
+benchmark: export COMPOSE_FILE ?= docker-compose.benchmark.yml
+benchmark: export COMPOSE_PROJECT_NAME = apicast-benchmark
+benchmark: export WRK_REPORT ?= $(IMAGE_TAG).csv
+benchmark: export DURATION ?= 300
+benchmark:
+	DURATION=$$(( $(DURATION) / 10 )) $(DOCKER_COMPOSE) run wrk
```
Comment (davidor) [inline Makefile]: Why / 10 ?

Comment (mikz) [reply Makefile]: A tenth of the full test for a warmup should be good no ?

Comment (davidor) [reply Makefile]: :+1:  . Can you please add `## warmup` there to make this clearer?

Comment (mikz) [reply Makefile]: Fixed.

**Pull request merged**